pkgdb2client/utils.py

```diff
@@ -255,26 +267,37 @@ def check_package_creation(info, bugid, pkgdbclient):
             messages["bad"].append(
                 'Non-packager {0} commented on review bug'.format(user))
 
+    bug_creator, bug_creator_full = get_fasinfo(bug.creator)
+    if bug_creator == requester:
+        messages["good"].append("Review bug created by requester `{0}`".format(
+            bug_creator_full))
+    else:
+        messages["bad"].append("Review bug created by `{0}` but request "
+                               "by {1}".format(bug_creator_full, requester))
+
     # Check who updated the fedora-review flag to +
     for flag in bug.flags:
         if flag['name'] == 'fedora-review':
             if flag['status'] == '+':
-                flag_setter = flag['setter']
+                flag_setter_email = flag['setter']
+                flag_setter, flag_setter_full = get_fasinfo(flag_setter_email)
                 if is_packager(flag_setter):
                     messages["good"].append(
-                        'Review approved by packager {0}'.format(flag_setter))
+                        'Review approved by packager `{0}`'.format(
+                            flag_setter_full))
```

Comment:
Should we include their username here also?  i.e.  'Review approved by packager `Ralph Bean (ralph)`'?